Doc/library/multiprocessing.rst

```diff
@@ -673,6 +673,15 @@ The :mod:`multiprocessing` package mostly replicates the API of the
 
       .. versionadded:: 3.7
 
+   .. method:: closed()
```

Comment:
```suggestion
   .. attribute:: closed
```

I think you should put this attribute after the documentation for the `close()` method.

Reply:
It doesn't seem to fit. It's actually a method, although there is a wrapper.

Reply:
For Sphinx, this matters. Because otherwise, people will think that it's callable (namely `p.closed()`) but it's not (it's `p.closed`).

Reply:
Hmmm, Seems to make sense.